db/seeds.rb

```diff
@@ -1,2 +1,11 @@
 AdminUser.create!(email: 'admin@example.com', password: 'password') if Rails.env.development?
 Setting.create_or_find_by!(key: 'min_version', value: '0.0')
+
+User.create(first_name: "Matias", last_name: "Cabrera", email: "matias@gmail.com", password: "12345678")
```

Comment:
we recommend to use single quotes for strings

Reply:
Received